packages/common/src/hooks/useDialog/useDialog.tsx

```diff
@@ -128,7 +128,7 @@ export const useDialog = (dialogProps?: DialogProps): DialogState => {
           sx={{ overflowX: 'hidden', ...contentSX }}
         >
           <Slide in={slideConfig.in} direction={slideConfig.direction}>
-            <div> {children}</div>
+            <div>{slideConfig.in && children}</div>
```

Comment:
Making whatever is sliding unmount (triggers auto focus 🤔 )

It does actually unmount regularly, but will remount just before the animation begins

Reply:
thanks - I wondered about that. was noticeable in request reqs